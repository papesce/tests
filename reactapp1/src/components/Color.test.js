import { shallow } from 'enzyme'
import Color from './Color'
import React from 'react'
import setupGlobals from "../../tests/global"

describe("<Color /> UI Component", () => {
    setupGlobals()
    it("Renders correct properties", () => {
        let output = shallow(
            <Color title="Test Color"
                color="#F0F0F0"
                rating={3}
                timestamp="Mon Apr 11 2016 12:54:19 GMT-0700 (PDT)"
            />
        ).html()

        expect(output).toMatchSnapshot()
    })

})
