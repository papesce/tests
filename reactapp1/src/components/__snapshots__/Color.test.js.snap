// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Color /> UI Component Renders correct properties 1`] = `"<section class=\\"color\\"><h1>Test Color</h1><button><svg fill=\\"currentColor\\" preserveAspectRatio=\\"xMidYMid meet\\" height=\\"1em\\" width=\\"1em\\" viewBox=\\"0 0 40 40\\" style=\\"vertical-align:middle\\"><g><path d=\\"m15.9 16.4v12.9q0 0.3-0.2 0.5t-0.5 0.2h-1.4q-0.3 0-0.5-0.2t-0.2-0.5v-12.9q0-0.3 0.2-0.5t0.5-0.2h1.4q0.3 0 0.5 0.2t0.2 0.5z m5.7 0v12.9q0 0.3-0.2 0.5t-0.5 0.2h-1.4q-0.3 0-0.5-0.2t-0.2-0.5v-12.9q0-0.3 0.2-0.5t0.5-0.2h1.4q0.3 0 0.5 0.2t0.2 0.5z m5.8 0v12.9q0 0.3-0.2 0.5t-0.6 0.2h-1.4q-0.3 0-0.5-0.2t-0.2-0.5v-12.9q0-0.3 0.2-0.5t0.5-0.2h1.4q0.4 0 0.6 0.2t0.2 0.5z m2.8 16.2v-21.2h-20v21.2q0 0.5 0.2 0.9t0.3 0.6 0.2 0.2h18.6q0.1 0 0.2-0.2t0.4-0.6 0.1-0.9z m-15-24h10l-1.1-2.6q-0.1-0.2-0.3-0.3h-7.1q-0.2 0.1-0.4 0.3z m20.7 0.7v1.4q0 0.3-0.2 0.5t-0.5 0.2h-2.1v21.2q0 1.8-1.1 3.2t-2.5 1.3h-18.6q-1.4 0-2.5-1.3t-1-3.1v-21.3h-2.2q-0.3 0-0.5-0.2t-0.2-0.5v-1.4q0-0.3 0.2-0.5t0.5-0.2h6.9l1.6-3.8q0.3-0.8 1.2-1.4t1.7-0.5h7.2q0.9 0 1.8 0.5t1.2 1.4l1.5 3.8h6.9q0.3 0 0.5 0.2t0.2 0.5z\\"></path></g></svg></button><div class=\\"color\\" style=\\"background-color:#F0F0F0\\"></div><div class=\\"time-ago\\">4/11/2016</div><div><div class=\\"star-rating\\"><div class=\\"star selected\\"></div><div class=\\"star selected\\"></div><div class=\\"star selected\\"></div><div class=\\"star\\"></div><div class=\\"star\\"></div><p>3 of 5 stars</p></div></div></section>"`;
